joomla/joomla-cms
# [4.0] Add period

### Summary of Changes
Add period to the alert message.


### Testing Instructions
Code review.

or 

Go to System > Update > Extensions
Click `Clear Cache` button.
See message.
